go.mod

```diff
@@ -14,10 +14,10 @@ require (
 	github.com/pkg/errors v0.9.0
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.16.4
+	k8s.io/api v0.17.0
 	k8s.io/apimachinery v0.17.0
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/component-base v0.16.4
+	k8s.io/component-base v0.17.0
```

Comment:
Nit: you can add a replace also for `k8s.io/component-base` to pin it to kubernetes-1.16.0.
```
	k8s.io/component-base => k8s.io/component-base v0.0.0-20190918160511-547f6c5d7090 // kubernetes-1.16.0
```

Reply:
done